pages/index.tsx

```diff
@@ -42,6 +43,14 @@ export default function Home() {
   return (
     <DarkMode>
       <Track>
+        <NextSeo
+          title="Thirdweb Authentication | thirdweb"
```

Comment:
adam plz stop with the capitalization of thirdweb :(

Reply:
also don't need `| thirdweb` since we already automatically append it for every title